### PR TITLE
WIP Bug 1430417 Add performance information with regards to IPsec 

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Comment by mburke5678** [comment]
@jeremyeder Do you have any information on the affect on performance when IPSec is enabled in OpenShift? As requested in:
https://bugzilla.redhat.com/show_bug.cgi?id=1430417

File: admin_guide/ipsec.adoc [hunk 1]
```diff
@@ -250,6 +250,11 @@ and begin encrypting:
 # systemctl enable ipsec
 ----
 
+[NOTE]
```
Comment (jeremyeder) [inline admin_guide/ipsec.adoc]: I think this duplicates what's already in paragraph 2 of the pre-req section https://github.com/openshift/openshift-docs/blob/master/admin_guide/ipsec.adoc#step-1-prerequisites

Comment (mburke5678) [inline admin_guide/ipsec.adoc]: It seems to, with the pre-req section having more detail. 
I can close this PR and leave the note out, if that makes sense.

**Comment by mburke5678** [comment]
Closing based on @jeremyeder comments above.

**Pull request closed without merging**